docs/getting-started/peoplefinder/understanding-policies.mdx

```diff
@@ -1,208 +1,111 @@
 ---
 sidebar_label: Understanding Policies
 title: Understanding Policies
-description: Deeper dive into the structure and content of Aserto policies
+description: Learn how Aserto policies work, how to author them, and how they are used in PeopleFinder.
 ---
 
 # Understanding Policies
 
-## The Structure of a Policy Bundle
+In this section you'll learn about Aserto Policies, and how they are used within PeopleFinder. You'll also see how to use the Evaluator in the Aserto Console to test out those Policies.
 
-The Aserto Authorizer is loaded with a policy bundle that includes all the assets needed to make decisions. The bundle includes a `policies` folder, a `data.json` file and a manifest file.
+## What is a Policy?
 
-In this example, the `data.json` file specifies the roles that are available for all the policies. For each role, it enumerates the module which governs them. Finally, for each module it enumerates the permissions settings that would apply for that role and module combination. Every module corresponds to a method and route in the API, following the convention `policyroot.METHOD.route` (where the route is dot delimited instead of slash delimited). For example:
+A Policy is a collection of decisions that govern access to the resources within an application. At runtime, Aserto uses Policies to determine whether or not to allow a caller to access resources.
 
-```json
-{
-  "roles": {
-    "viewer": {
-        "description": "PeopleFinder viewer",
-        "perms": {
-          "peoplefinder.GET.api.users": {
-              "allowed": true,
-              "visible": true,
-              "enabled": true
-          },
-         ...
-        }
-    },
-    "editor": {...},
-    "admin": {...}
-  }
-}
-```
+## How Aserto Policies are authored and deployed
 
-In this snippet we can see that people with the role of `viewer` are `allowed` to access the method **GET** and the route `/api/users` for the policy root `peoplefinder`.
+Aserto Policies contain a collection of Policy Modules which are authored in a declarative language called [Rego](https://www.openpolicyagent.org/docs/latest/policy-language/). For each API that needs to be authorized, there should be a corresponding Policy Module. Each module contains one or more decisions. All the modules within your application are packaged into a Policy Image that can then be deployed to a Policy Instance, which uses an [Aserto Authorizer](/docs/overview/authorizer) behind the scenes at runtime.
```

Comment:
into a Policy Image that can be bound to a Policy Instance.  Every Policy Instance has a hosted [Aserto Authorizer](/docs/overview/authorizer) instance automatically running, so you can easily integrate your application and test it against a running authorizer without having to deploy anything. 

When it comes to deploy your application, you can run an authorizer instance close to your application (either as a sidecar or as another microservice), so that you're not dependent on a hosted authorizer in production.